sdk/src/helpers.ts

```diff
@@ -27,6 +27,12 @@ export function buildProjectQuery(options?: EmbedOptions) {
       params.push(`file=${options.openFile}`);
     }
 
+    if (Array.isArray(options.openFile)) {
+      options.openFile.forEach((file) => {
+        params.push(`file=${file}`);
```

Comment:
```suggestion
        if (typeof file === 'string') {
          params.push(`file=${file}`);
        }
```

Reply:
I think it's a good point, @fvsch. I would retire this section in favor of the developer.stackblitz site – we already treat it as the primary source of documentation, and maintaining both doesn't seem to have much upside.